### PR TITLE
Bugfix: some bug fix.

### PR DESCRIPTION
**Pull request by champly**
Signed-off-by: champly <champly@outlook.com>

- some args not init.
- klog init fix.
- add proxy-agent `cluster-proxy-open-cluster-management.io-proxy-agent-signer-client-cert` secret.

### CODE REVIEW

File: controllers/clustermanagementaddon_controller.go [hunk 1]
```diff
@@ -63,6 +63,8 @@ func RegisterClusterManagementAddonReconciler(
 				EventRecorder: events.NewInMemoryRecorder("ClusterManagementAddonReconciler"),
 			}
 		},
+		SecretLister:     secretInformer.Lister(),
+		SecretGetter:     nativeClient.CoreV1(),
```
Comment (yue9944882) [inline controllers/clustermanagementaddon_controller.go:67]: 👍

File: pkg/addon/agent/agent.go [hunk 2]
```diff
@@ -84,9 +84,20 @@ func (p *proxyAgent) Manifests(managedCluster *clusterv1.ManagedCluster, addon *
 		proxyServerLoadBalancer = entrySvc
 	}
 
+	// get proxy client secret
+	proxyClientSecret, err := p.nativeClient.CoreV1().
```
Comment (yue9944882) [inline pkg/addon/agent/agent.go:88]: can you elaborate the case why we need to persist the proxy-client secret to the managed clusters? is that for communication between the managed clusters? anyway, i think we should separate this change into another pull for further expansion, thx

Comment (champly) [reply pkg/addon/agent/agent.go:88]: i will split the submission of secret in a moment. this secret is for communication between the `proxy-agent` and the `proxy-server`.

**Review by yue9944882 — COMMENTED** [review]
/lgtm
/approve

**Comment by yue9944882** [comment]
@champly can you sign off your commit by `git reset --soft origin/main; git commit -s -m '<your message>'`?

**Comment by champly** [comment]
> @champly can you sign off your commit by `git reset --soft origin/main; git commit -s -m '<your message>'`?

Add Signed. PTAL

**Review by yue9944882 — COMMENTED** [review]
/lgtm

**Pull request merged**